src/lib/ui/device/OnlineStatus.tsx

```diff
@@ -36,15 +36,15 @@ function AccountOnlineContent(props: { isOnline: boolean }): JSX.Element {
   if (isOnline) {
     return (
       <>
-        <OnlineStatusOnlineIcon />
+        <OnlineStatusOnlineIcon style={{ fontSize: '24px' }} />
```

Comment:
I was not 100% sure using `style` here is safe. Seam Components must support strict content security policy. Some `style` rules are ok to use because they use a different browser API (I cannot find the reference at the moment).

Anyway, I [tested this in the preview env](https://react-162orvajj.seam.vc/examples/basic/) to see if it triggers CSP errors and it seems ok. Maybe because it's a svg style? 

Just something to keep in mind when using `style` (we should always use css where possible, but I'm guessing this is not a case where it is possible).

Reply:
Oh  I'll create a className then. Style can be omitted